config/application.rb

```diff
@@ -9,7 +9,7 @@ class Application < Rails::Application
     config.encoding = "utf-8"
     config.filter_parameters += [:password]
     config.active_support.escape_html_entities_in_json = true
-
+    config.autoload_paths += %W["#{config.root}/app/validators/"]
```

Comment:
jj, didn´t know where to place it. I have removed the path, you are right, no need to add it to the path, it´s already there.